e2e-pw/scripts/check-flaky.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
nice! is there a way to avoid sh files? hard to maintaim IMO

Reply:
maybe a python script can do similar task

Reply:
I totally agree, but since the script is just calling a command and checking its exit code, I think its best served in bash. (it'd arguably be more complicated in python with all the ceremony of `subprocess.run`s) It looks a little overwhelming because I added argparse and colors, but if you strip that away, it's a super simple script. Furthermore, I doubt this will ever change, so I don't worry about the maintainability here.